src/contentscript/overlay/root/components/Notification/index.tsx

```diff
@@ -26,7 +26,8 @@ export interface NotificationProps {
 export const Notification: React.FC<NotificationProps> = (props) => {
   const { icon, title, date, onClear, _id, description, href, status, actions, stateNotify } = props
 
-  const [isDelete] = useState(false)
+  // const [isDelete] = useState(false)
```

Comment:
Why do you not just delete but comment this parameter and related lines? Is it have a reason?